docker/builder/beaver.Dockerfile

```diff
@@ -0,0 +1 @@
+FROM sociomantictsunami/dlang:v4
```

Comment:
Why not latest?

Reply:
will update

Reply:
Updated to `v6` as it is used in `docker/Dockerfile.dmqnode`